src/frontend/mod.rs

```diff
@@ -146,6 +146,32 @@ impl<'a, B: Backend<'a>> Frontend<'a, B> {
         }
     }
 
+    fn convert_text(&mut self, mut text: Cow<'a, str>) {
+        // test if the text contains a latex command
+        lazy_static! {
+            // latex actually isn't really parsable, so this is only a very rough estimate of common commands
+            // Expect: \commandname<arg>[arg]{arg}{arg}...
+            static ref LATEX_COMMAND: Regex = Regex::new(r"\\[a-zA-Z]+(?:$|<.*?>|\[.*?\]|\{.*?\})").unwrap();
```

Comment:
What about the regex at https://github.com/oberien/heradoc/blob/548612591ffd0ac58bd9a56e042d85da34e4ac9f/src/backend/latex/simple.rs#L32? I'm a big confused about the purpose of both, and maybe an explanation in the docu might be good for visually parsing both and mentioning their individual limitations.

Reply:
The regex in the text-generator is removed in this PR. It was used to detect possible latex command to prevent escaping `\\` before the command. It was pretty sophisticated and should detect most beginnings of latex commands. It would probably have a lot of false positives.

The new regex used in the frontend is a very rough estimate and won't catch most special command syntaxes. It should catch any command of the form `\name<args>[args]{args}`, but will probably have some false negatives.